_extras/customization.md

```diff
@@ -163,17 +230,32 @@ please follow the steps bellow:
 
 1.  Add the workshop-template repository as upstream:
 
-        $ git remote add upstream https://github.com/carpentries/workshop-template.git
+~~~
+$ git remote add upstream https://github.com/carpentries/workshop-template.git
+~~~
+{: .language-bash}
 
 2.  Fetch the data from upstream repository (also know as the workshop-template
     repository):
 
-        $ git pull upstream
+~~~
+$ git pull upstream
+~~~
+{: .language-bash}
 
 4.  Address possible merge conflicts, and
 
-        $ git commit -a
+~~~
+$ git commit -a
+~~~
+{: .language-bash}
 
 5.  Push the changes to your repository on GitHub:
 
-        $ git push origin gh-pages
+~~~
+$ git push origin gh-pages
+~~~
+{: .language-bash}
+
+
```

Comment:
Currently, this list appears as
```
1. ...

1. ...

1. ...
```

I submitted #674 against this branch to fix it.